### ISSUE TITLE
Deploy as lambda infront of new regional notes

### ISSUE THREAD

**Issue by nick-verida** [verida/storage-node#15]

**Comment by nick-verida** [comment]
Testing locally (`yarn run test`) when configured to use 

```
"DB_HOST" : "dbc1.us.testnet.verida.tech",
"DB_PORT" : "5984",
```

times out. 

Added logging in `userManager.create()` like this:

```
    let usersDb = couch.db.use('_users');
    try {
      return await usersDb.insert(userData);
    } catch (err) {
      console.log(err)
      this.error = err;
      return false;
    }
```

I get 

```
Error: connect ETIMEDOUT 44.197.31.12:5984
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1148:16) {
  scope: 'socket',
  errid: 'request',
  errno: -60,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '44.197.31.12',
  port: 5984,
  description: 'connect ETIMEDOUT 44.197.31.12:5984',
  stacktrace: [
    'Error: connect ETIMEDOUT 44.197.31.12:5984',
    '    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1148:16)'
  ]
}
```

Note that 44.197.31.12 is not dbc1.us.testnet.verida.tech

**Comment by tahpot** [comment]
TLDR;

The load balancer was configured with three subnets. One of the subnets was a private subnet, not public. Updating to use the public subnet resolved this issue.

---

I can replicate the error when talking to `dbc1.us.testnet.verida.tech`.

Interestingly the user record is actually created in the cluster.

I have connected to all three clusters in the node and verified they can all see each other. All three return the same result for [/_membership](http://ec2-34-237-1-39.compute-1.amazonaws.com:5984/_membership):

```js
{
  "all_nodes": [
    "couchdb@34.203.40.120",
    "couchdb@34.237.1.39",
    "couchdb@52.201.249.230"
  ],"cluster_nodes":[
    "couchdb@34.203.40.120",
    "couchdb@34.237.1.39",
    "couchdb@35.173.190.65",
    "couchdb@52.201.249.230",
    "couchdb@ec2-34-203-40-120.compute-1.amazonaws.com",
    "couchdb@ec2-34-237-1-39.compute-1.amazonaws.com"
  ]
}
```

There is no mention of `44.197.31.12` IP address.

These are the nodes in the cluster:

- us-east-1a (ec2-52-201-249-230.compute-1.amazonaws.com)
- us-east-1b (ec2-34-237-1-39.compute-1.amazonaws.com)
- us-east-1c (ec2-34-203-40-120.compute-1.amazonaws.com)
 
I'm not sure where `35.173.190.65` is coming from.

`44.197.31.12` is not in our list of elastic IP addresses in AWS testnet account for either US or SG regions.

```
ping dbc1.us.testnet.verida.tech
PING couchdb-cluster-us-east-1-1541724271.us-east-1.elb.amazonaws.com (44.197.31.12): 56 data bytes
```

`44.197.31.12` must be the IP address of the load balancer.

Upon deeper investigation the load balancer was configured with three subnets. One of the subnets was a private subnet, not public. Updating to use the public subnet resolved this issue.
